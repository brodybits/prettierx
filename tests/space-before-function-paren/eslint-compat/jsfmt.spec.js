run_spec(__dirname, ["babel", "babel-flow", "flow", "typescript"], {
  spaceBeforeFunctionParen: true
});
run_spec(__dirname, ["babel", "babel-flow", "flow", "typescript"]);
