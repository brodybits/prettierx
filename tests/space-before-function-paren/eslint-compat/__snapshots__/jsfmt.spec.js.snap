// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`eslint.js 1`] = `
====================================options=====================================
parsers: ["babel", "babel-flow", "flow", "typescript"]
printWidth: 80
spaceBeforeFunctionParen: true
                                                                                | printWidth
=====================================input======================================
/*eslint space-before-function-paren: "error"*/
/*eslint-env es6*/

function foo() {
    // ...
}

var bar = function() {
    // ...
};

var bar = function foo() {
    // ...
};

class Foo {
    constructor() {
        // ...
    }
    bar() {
        // ...
    }
}

var foo = {
    bar() {
        // ...
    }
};

var foo = async() => 1

=====================================output=====================================
/*eslint space-before-function-paren: "error"*/
/*eslint-env es6*/

function foo () {
  // ...
}

var bar = function () {
  // ...
};

var bar = function foo () {
  // ...
};

class Foo {
  constructor () {
    // ...
  }
  bar () {
    // ...
  }
}

var foo = {
  bar () {
    // ...
  }
};

var foo = async () => 1;

================================================================================
`;

exports[`eslint.js 2`] = `
====================================options=====================================
parsers: ["babel", "babel-flow", "flow", "typescript"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
/*eslint space-before-function-paren: "error"*/
/*eslint-env es6*/

function foo() {
    // ...
}

var bar = function() {
    // ...
};

var bar = function foo() {
    // ...
};

class Foo {
    constructor() {
        // ...
    }
    bar() {
        // ...
    }
}

var foo = {
    bar() {
        // ...
    }
};

var foo = async() => 1

=====================================output=====================================
/*eslint space-before-function-paren: "error"*/
/*eslint-env es6*/

function foo() {
  // ...
}

var bar = function() {
  // ...
};

var bar = function foo() {
  // ...
};

class Foo {
  constructor() {
    // ...
  }
  bar() {
    // ...
  }
}

var foo = {
  bar() {
    // ...
  }
};

var foo = async () => 1;

================================================================================
`;
